spec/Binary/20-BINARY_FORMAT.md

```diff
@@ -1,3 +1,54 @@
 # Binary format
 
-TBD
\ No newline at end of file
+Binary format document describes how to encode each field - `traceparent` and
+`tracestate`.
+
+# `Traceparent` binary format
+
+The field `traceparent` encodes the version of the protocol and fields
+`trace-id`, `span-id` and `recorded`. Each field starts with the one byte field
+identifier with the field value following immediately after it. Field
+identifiers are used as markers for additional verification of the value
+consistency and may be used in future for the versioning of the `traceparent`
+field.
+
+``` abnf
+traceparent     = version "0" trace-id "1" parent-id "2" recorded
+trace-id        = 16BYTES
+parent-id       = 8BYTES
+trace-options   = 1BYTE   ; only the least significant bit is counted
+```
+
+Unknown field identifier (anything beyond `0`, `1` and `2`) should be treated as
+invalid `traceparent`. All zeroes in `trace-id` and `parent-id` invalidates the
+`traceparent` as well.
+
+## `Traceparent` example
+
+``` js
+{0,
+  0, 75, 249, 47, 53, 119, 179, 77, 166, 163, 206, 146, 157, 0, 14, 71, 54,
+  1, 52, 240, 103, 170, 11, 169, 2, 183,
+  2, 1}
+```
+
+This corresponds to:
+
+- `traceId` is
```

Comment:
Consider unifying how fields are referenced (`trace-id` vs `traceId` etc)

Reply:
oh, thanks for noticing